examples/nest/tsodyks_depressing.sli

```diff
@@ -21,7 +21,7 @@
  */
 
  /* BeginDocumentation
-Name: test_tsodyks_depressing - sli script for overall test of iaf_neuron model
+Name: test_tsodyks_depressing - sli script for overall test of iaf_psc_exp model
```

Comment:
Actually, this example (and pretty much the other three tsodyks_* examples in examples/nest) are not in good shape. I suggest we remove all four sli examples here, we have proper Python examples for Tsodyks-style facilitation and depression.